plugin.d.ts

```diff
@@ -39,7 +39,11 @@ declare module 'fastify' {
      */
     unsignCookie(
       value: string,
-    ): string | false;
+    ): {
+      valid: boolean;
+      renew: boolean;
+      value: string | false;
```

Comment:
`value` is either string or `null` in v5.
It's never `false`.

Reply:
thats true, missed the line 21